mapit/views/areas.py

```diff
@@ -394,8 +394,15 @@ def areas_by_point(request, srid, x, y, bb=False, format=''):
 
     q = query_args(request, format)
 
+    try:
+        within = float(request.GET.get('within', 0))
```

Comment:
We should validate that this is >= zero, as negative values appear to cause an `InternalError: Tolerance cannot be less than zero` SQL error. Might be an idea to clamp it to an upper bound too, I'm not sure if a really high figure could otherwise be used to get every single area and if that would be a problem?

Reply:
I'll make it a config variable so we can adjust as necessary.